sql/log_event.cc

```diff
@@ -3149,7 +3149,7 @@ Rows_log_event::Rows_log_event(const uchar *buf, uint event_len,
   uchar *ptr_after_width= (uchar*) ptr_width;
   DBUG_PRINT("debug", ("Reading from %p", ptr_after_width));
   m_width= net_field_length(&ptr_after_width);
-  DBUG_PRINT("debug", ("m_width=%lu", m_width));
+  DBUG_PRINT("debug", ("m_width=%u", m_width));
```

Comment:
But https://github.com/MariaDB/server/blob/fe3432b3bd5be831ed2a6071e2d8e556fe8893ba/sql/log_event.cc#L500
I forgot where this change came from.